Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -169,6 +212,18 @@ extension SettingsDictionary {
     public func debugInformationFormat(_ format: DebugInformationFormat) -> SettingsDictionary {
         merging(["DEBUG_INFORMATION_FORMAT": SettingValue(format)])
     }
+
+    // MARK: - Info.plist Values
+
+    /// Sets `INFOPLIST_KEY_LSApplicationCategoryType` to the given category.
+    public func applicationCategory(_ category: ApplicationCategory) -> SettingsDictionary {
+        merging(["INFOPLIST_KEY_LSApplicationCategoryType": SettingValue(category.rawValue)])
```

Comment:
Shouldn't this just be 
```suggestion
        merging(["LSApplicationCategoryType": SettingValue(category.rawValue)])
```
?
If we want it in the plist we should change the PLIST helper instead

Reply:
👍 Tuist currently generates an independent `Info.plist` file, as such worth keeping the plist keys definitions consistent rather than mixing the setting style and plist file style.

Perhaps an independent exploration can be done to see how to add an option to switch between both, and what the limitations of the either approach is (e.g. are there keys that can only be defined in one but not the other? e.g. custom keys?)

Reply:
As per the documentation of this key found [here](https://developer.apple.com/documentation/xcode/build-settings-reference#Application-Category) this has no impact on the `.plist`. If the user sets `GENERATE_INFOPLIST_FILE` to `YES` then at that point it should be reflected in the `.plist`.

I assume what you guys mean is that now if the user opt to add `GENERATE_INFOPLIST_FILE` this won't be reflected in the `.plist` generated by Tuist.

We can:

1. Document this and say that this will be added to the `Identity section` but not the `.plist` even with `GENERATE_INFOPLIST_FILE` set to `YES` and then work on it in a separate PR (I can start with it this week)
2. Delete the whole PR altogether 🥲

If there is anything else I can clear up please let me know.

Reply:
My point is that, given we are not supporting the GENERATE_INFOPLIST_FILE in any other way with any other helper, I wouldn' add this helpers here, but I would rather add them to [the current way of defining the plist](https://github.com/tuist/tuist/blob/main/Sources/ProjectDescription/InfoPlist.swift), probably as an extension on dictionaries of [String: InfoPlist.Value]?

Reply:
I will create a fresh project (without tuist) play around with those values, see how everything changes, document it and then we can make a better decision. I'll keep you guys posted.

Reply:
Creating a new iOS project (without Tuist):
- `GENERATE_INFOPLIST_FILE` is by default set to `YES`.
-  `INFOPLIST_KEY_LSApplicationCategoryType` adds nothing to the `.plist` file. But modifies the Identity tab under General. And in the `.pbxproj` adds `INFOPLIST_KEY_LSApplicationCategoryType = "public.app-category.education";` under `buildSettings` for Release and Debug.

There is a better explanation on the new management system found here: https://juejin.cn/post/7197361396219772983
Page is in Chinese but I translated the page to English.

Copied form that article: 

```
Build Settings - Generate Info.plist File (GENERATE_INFOPLIST_FILE) This field is newly introduced by Xcode13, it indicates whether to enable the new mode of generating Info.plist file, YES starts, NO does not enable. This field defaults to YES for new projects, and NO for old projects.

When the value is YES, Xcode uses the new Info.plist file generation mode. Xcode will take data from the Build Setting - Info.plist Values ​​module and Info - Custom iOS Target Properties to generate the final Info.plist file when packaging .
When the value is NO, Xcode uses the old Info.plist file generation mode. We only need to configure parameters in the Info.plist physical file in the project as before, and Xcode will read the configuration in the Info.plist physical file to generate the Info.plist file in the final package body when packaging.

First of all, let's make it clear: the name of the App is finally determined by the CFBundleDisplayName field in the Info.plist file in the compiled package body . If there is no CFBundleDisplayName field in the Info.plist, the value of the CFBundleName field will be taken.
Due to the new features of Xcode14, the Display Name set in the Xcode panel will only be synchronized to Build Setting - Info.plist Values ​​- Bundle Display Name .
Since we used the old Info.plist file mode (that is, the value of BuildSetting - Generate Info.plist File is NO), the settings of BuildSetting - Info.plist Values ​​- Bundle Display Name will not be synchronized to the final generated package body Info.plist file. This indirectly leads to the problem that setting the Display Name in the Xcode panel does not take effect.
```

So basically long story short this is a whole new `.plist` management system.

I assume this PR is a half solution, and should be addressed in a bigger way?
What would be the best next steps then?

Reply:
Discussing [here](https://tuistapp.slack.com/archives/CCXC295QS/p1678896591526059?thread_ts=1678697609.654359&cid=CCXC295QS)